src/libs/tools/tests/testtool_backendbuilder.cpp

```diff
@@ -93,6 +93,8 @@ TEST (MountBackendBuilder, basicAddRem)
 	EXPECT_TRUE (bb.validated ());
 }
 
+// ASAN reports memory leaks for the following test: https://travis-ci.org/sanssecours/elektra/jobs/418573941
+#ifndef ENABLE_ASAN
```

Comment:
This is dangerous (we might forget which tests are included).

Could you please instead add a DISABLED_ prefix to the test name (simply do a ## concat in the case of ENABLE_ASAN). Then there will be a clear message that not all tests have been run.